### ISSUE TITLE
Improvement: Add ability to Export Workbench Project to Single Machine Config File

### ISSUE THREAD

**Issue by PeteBlanchard** [silk-framework/silk#50]
When exporting a workbench project it creates a project zip file (so you can import on another machine).  It would be nice to be able to export it as a named config file that can be passed to the singlemachine variant as the formats are (understandably) very different.

**Comment by robertisele** [comment]
Do you want to export a XML file according to the Silk Link Specification Language [1]? You can do this if you go to the linkage rule editor and click on "Get Linkspec". The resulting link spec should be executable in Silk SingleMachine.

[1] https://github.com/silk-framework/silk/blob/master/doc/LinkSpecificationLanguage.md

**Comment by PeteBlanchard** [comment]
Cool, thanks.  I did not know that.  That is good and helpful.  But I would have to do that on a per linkage basis...if I have several defined and want to run them all...I would want the project.  :-)

Of course, I could do them individually and just add the Interlinks...

**Comment by robertisele** [comment]
You are right, exporting a link spec that contains all tasks in the project is currently not possible, but would be helpful. I have marked this issue as enhancement, so it can be done as soon as there is time.
